test/inspect-provenance.test.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { test } from 'tap';
+import { chdirWorkspaces, activateVirtualenv } from './test-utils';
+
+import pluginImpl = require('../lib');
+
+const pipAppExpectedDependenciesOnlyProvenance = {
```

Comment:
where is provenance calculated? This PR did not mention anything about it, is this a new feature in this PR?

Reply:
It was added in the previous PR, using "--only-provenance" flag clearly visible in this test and documented in the Python code.